gp-templates/project-import.php

```diff
@@ -24,14 +24,22 @@
 
 gp_title( $gp_title );
 gp_tmpl_header();
-?>
 
+$format_extensions = array();
+foreach ( GP::$formats as $format ) {
+	$format_extensions = array_merge( $format_extensions, $format->get_file_extensions() );
+}
+// Remove redundant alternate extensions that have the same ending and add leading '.'.
+$format_extensions = array_unique( preg_replace( '/^(\w*\.)*/', '.', $format_extensions ) );
```

Comment:
Would this `preg_replace` be better suited to be inside `get_file_extensions()`?

Reply:
Moving the `preg_replace` inside the `get_file_extensions()` will remove these from the output: `jed.json`, `resx.xml`, `ngx.json`.
I'd guess this won't be desirable and might break it's use elsewhere.

Reply:
Yes, you are right. I proposed in the review comment to add a new function.